public/404.html

```diff
@@ -1,67 +1,213 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>The page you were looking for doesn't exist (404)</title>
+  <title>Page Not Found | Classroom</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link href='//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
```

Comment:
> stylesheet_link_tag may allow you to load app styles here. If not, we might just have to hardcode.

I don't think there is an easy way out to add assets pipeline in Public HTML pages without using Dynamic Error Pages(correct me if I'm wrong). I have used 100% inline styles as I didn't find any easy way out to add assets pipeline to it. Maybe I am unaware of the superior way to handle it unless we precompile HTML error pages like [this](https://www.icelab.com.au/notes/precompiled-rails-static-404-and-500-pages/).